src/cli/api_client.rs

```diff
@@ -237,6 +238,7 @@ impl ApiClient {
                 on_chain_address,
             )),
             satoshis,
+            metadata: Some(serde_json::from_value(metadata).unwrap()),
```

Comment:
https://github.com/GaloyMoney/bria/pull/121/files#r1170344951

Reply:
Definitely use `from_value(_)?`.

Read up on `?` syntactic sugar if you don't know it.